spec/helpers/application_helper_spec.rb

```diff
@@ -2,20 +2,16 @@
 
 require 'rails_helper'
 
-# Specs in this file have access to a helper object that includes
-# the ProduceItemsHelper. For example:
-#
-# describe ProduceItemsHelper do
-#   describe "string concat" do
-#     it "concats two strings with spaces" do
-#       expect(helper.concat_strings("this","that")).to eq("this that")
-#     end
-#   end
-# end
 RSpec.describe ApplicationHelper, type: :helper do
   describe 'formatted month name' do
     it 'changes the month_index to a month_name' do
       expect(helper.formatted_month_name(1)).to eq('janeiro')
     end
   end
+
+  describe 'formatted month short' do
```

Comment:
**suggestion:** I meant literally `#formatted_month_short`, since you are describing the method. In Ruby notation, `#` indicates instance methods and `.` or `::` indicates class methods. Using the following as example:

```rb
class MyExampleClass
  def self.a_class_method; end
  
  def an_instance_method; end
end
```

We would use the following to refer to them:
`MyExampleClass.a_class_method` or `MyExampleClass::a_class_method`
`MyExampleClass#an_instance_method`

You can see a clear example in the [Ruby docs](https://ruby-doc.org/core-3.1.0/Array.html). For that reason, I prefer to follow that convention in tests, after all you are describing a method.

Reply:
Ok! Then I'll also change `formatted month name` to `#formatted_month_name`. (I'll also checkout the docs)